.changes/v3.9.0/984-improvements.md

```diff
@@ -1,2 +1,3 @@
 * `vcd_external_network_v2` allows setting DNS fields `dns1`, `dns2` and `dns_suffix` for NSX-T
   backed entities so that it can be inherited by direct Org VDC networks  [GH-984]
+* `vcd_org_group` adds `OAUTH` as an option to argument `provider_type`  [GH-1013]
```

Comment:
If you could create a new file `1013-improvements.md` with this line.

We have this changelog format to avoid constant conflicts when making PRs :) https://github.com/vmware/terraform-provider-vcd/blob/main/.changes/README.md

Reply:
Sure. No problem.
The file numbering now makes sense to me. :)